<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdTermMainTerminal</class>
 <widget class="QMainWindow" name="PdTermMainTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PdTermMainTerminal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
    background-color: black;   /* Fundo preto */
    color: lime;              /* Letra verde vibrante */
    font-family: Consolas;    /* Fonte monoespaçada (opcional) */
    font-size: 12pt;          /* Tamanho da fonte (opcional) */
}</string>
    </property>
    <property name="backgroundVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConectar">
    <property name="title">
     <string>Conexão</string>
    </property>
    <addaction name="actionConectar"/>
    <addaction name="actionDesconectar"/>
    <addaction name="actionConfigura_o"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionteste"/>
    <addaction name="separator"/>
    <addaction name="actionPdTermPro_V1"/>
    <addaction name="actionDesenvolvido_por_PdSilva"/>
    <addaction name="actionLicen_a_GPLv3"/>
    <addaction name="actiongithub_com_ppsilv_pdtermpro"/>
    <addaction name="separator"/>
    <addaction name="action_Exit_2"/>
   </widget>
   <widget class="QMenu" name="menuLimpar">
    <property name="title">
     <string>Ferramentas</string>
    </property>
    <addaction name="action_Limpar"/>
    <addaction name="action_Enviar_arquivo"/>
   </widget>
   <addaction name="menuConectar"/>
   <addaction name="menuLimpar"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionPdTermPro_V1">
   <property name="text">
    <string>PdTermPro V1</string>
   </property>
  </action>
  <action name="actionDesenvolvido_por_PdSilva">
   <property name="text">
    <string>Desenvolvido por PdSilva</string>
   </property>
  </action>
  <action name="actionLicen_a_GPLv3">
   <property name="text">
    <string>Licença: GPLv3</string>
   </property>
  </action>
  <action name="actiongithub_com_ppsilv_pdtermpro">
   <property name="text">
    <string>github.com/ppsilv/pdtermpro</string>
   </property>
  </action>
  <action name="actionEnviar">
   <property name="text">
    <string>Enviar</string>
   </property>
  </action>
  <action name="action_Exit_2">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Limpar">
   <property name="text">
    <string>&amp;Limpar           Ctrl+L</string>
   </property>
  </action>
  <action name="action_Enviar_arquivo">
   <property name="text">
    <string>&amp;Enviar arquivo   Ctrl+S</string>
   </property>
  </action>
  <action name="actionConectar">
   <property name="text">
    <string>Conectar           Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDesconectar">
   <property name="text">
    <string>Desconectar    Ctrl+U</string>
   </property>
  </action>
  <action name="actionteste">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actiontsdfasd">
   <property name="text">
    <string>tsdfasd</string>
   </property>
  </action>
  <action name="actionConfigura_o">
   <property name="text">
    <string>Configuração</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
